src/lib/schematics/tree/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts

```diff
@@ -91,23 +94,23 @@ export class <%= classify(name) %>Component {
     };
   }
 
- /** Get the level of the node */
-  getLevel(node: TreeNode) {
+  /** Get the level of the node */
+  getLevel(node: FlatTreeNode) {
     return node.level;
   }
 
   /** Get whether the node is expanded or not. */
-  isExpandable(node: TreeNode) {
+  isExpandable(node: FlatTreeNode) {
     return node.expandable;
   };
 
+  /** Get whether the node has children or not. */
+  hasChild(index: number, node: FlatTreeNode){
```

Comment:
can we have a space before `{`?

Reply:
Good catch. Note that I just moved that already existing code one block up.